lighthouse-core/test/audits/script-treemap-data-test.js

```diff
@@ -72,6 +74,36 @@ describe('ScriptTreemapData audit', () => {
     });
   });
 
+  describe('coursehero fixture', () => {
+    /** @type {import('../../audits/script-treemap-data.js').TreemapData} */
+    let treemapData;
+    beforeAll(async () => {
+      const context = {computedCache: new Map()};
+      const {map, content} = loadSourceMapFixture('coursehero-bundle-1');
```

Comment:
does this fixture capture the `length of undefined` bug?

Reply:
what?

Reply:
I'm asking if this fixture gets hit by the bugs you are fixing here. i.e. without your logic changes does this test throw and/or have invalid results?

Reply:
(I think it does based on your PR description, but I couldn't tell what exactly happens in this test if you revert the changes)

Reply:
b/c of the `JsUsage: {}`, before this PR the audit would bail out before processing the source map. now it doesn't.